src/irmin-pack/unix/gc_worker.ml

```diff
@@ -118,6 +135,10 @@ module Make (Args : Gc_args.S) = struct
     (* Step 1. Open the files *)
     [%log.debug "GC: opening files in RO mode"];
     let stats = ref (Gc_stats.Worker.create "open files") in
+    let () =
+      report_old_file_sizes ~root ~generation:(generation - 1) stats |> ignore
```

Comment:
Dealing with newies will go away with the chunk-based GC so my thought is to leave this as-is.

Reply:
The handling of the suffix by the worker will go away, but not the stats. The newies is a measure of how much the main works during the GC, I would like this measure to be accurately displayed in the stats.

Could you find a way to improve that?

Reply:
The alternative I'm suggesting is to compute the exact size of the suffix in the main, before starting the Gc process, and then pass it as argument to the subprocess. Do you have other suggestions?

Reply:
I've just realized that the way I've built `Stats.Latest_gc.worker.files @ "new suffix"` is wrong for a similar reason, it doesn't contain the newies that will be appended by the finalise.

Another maybe-better solution would be to move the file sizes from `Stats.Latest_gc.worker` to `Stats.Latest_gc.stats`, this would then be the responsibility of the main to compute the size of the files.

Reply:
One more solution, let's remove the new suffix from `Stats.Latest_gc.worker.files` and let's not add the old suffix. The stats already contain some offsets, I'll use these from the benchmark.

```
    before_suffix_start_offset : int63;
    before_suffix_end_offset : int63;
    after_suffix_start_offset : int63;
    after_suffix_end_offset : int63;
```

Reply:
Sorry for not coming back to this sooner. 

I removed the old suffix, but we need the new suffix to read in finalise gc https://github.com/mirage/irmin/blob/main/src/irmin-pack/unix/gc.ml#L262 to transfer the latest newies.

Reply:
Thanks